.github/workflows/on-push-to-main-branch.yml

```diff
@@ -41,8 +41,7 @@ jobs:
         with:
           token: ${{ secrets.MOMENTO_MACHINE_USER_GITHUB_TOKEN }}
           release-type: python
-          default-branch: main
-          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false}]'
+          manifest-file: .release-please-manifest.json
```

Comment:
I don't think this is the root cause: `manifest-file` defaults to this already ("manifest-file | Path to the release-please versions manifest. Defaults to .release-please-manifest.json"). Also, I added `momento/__init__.py` as an extra file to modify and it did that